test/readme.md

```diff
@@ -0,0 +1,23 @@
+# Test instruction
+
+For the testing tools, we use both nodeunit and [chai-passport-strategy](https://github.com/jaredhanson/chai-passport-strategy). Nodeunit is used for general testing purpose where the passport framework is not involved, and chai-passport-strategy is used for the passport strategy workflow testing. Instructions on how to use chai-passport-strategy can be found [here](https://github.com/jaredhanson/chai-passport-strategy/blob/master/README.md).
+
+All the test files should have a *_test suffix in their names and be placed in the correct subdirectory, depending on the testing tools used. The following is the rule: 
```

Comment:
Looks like the italics go too far. I can't tell if that's just a GitHub editor bug though.